security.rst

```diff
@@ -4,144 +4,259 @@
 Security
 ========
 
-.. admonition:: Screencast
-    :class: screencast
+Symfony provides many tools to secure your application. Some HTTP-related
+security tools, like :ref:`secure cookies` and :ref:`CSRF protection` are
+provided by default. The SymfonySecurityBundle, which you will learn about
+in this guide, provides all authentication and authorization features
+needed to secure (parts of) your application.
 
-    Do you prefer video tutorials? Check out the `Symfony Security screencast series`_.
+.. _security-installation:
 
-Symfony's security system is incredibly powerful, but it can also be confusing
-to set up. Don't worry! In this article, you'll learn how to set up your app's
-security system step-by-step:
+To get started, make sure you have installed the SecurityBundle:
 
-#. :ref:`Installing security support <security-installation>`;
+.. code-block:: terminal
 
-#. :ref:`Create your User Class <create-user-class>`;
+    $ composer require symfony/security-bundle
 
-#. :ref:`Authentication & Firewalls <security-yaml-firewalls>`;
+If you have :ref:`Symfony Flex <symfony-flex>` installed, this also
+installs a ``security.yaml`` configuration file for you:
+
+.. code-block:: yaml
+
+    # config/packages/security.yaml
+    security:
+        # https://symfony.com/doc/current/security/experimental_authenticators.html
+        enable_authenticator_manager: true
+        # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
+        providers:
+            users_in_memory: { memory: null }
+        firewalls:
+            dev:
+                pattern: ^/(_(profiler|wdt)|css|images|js)/
+                security: false
+            main:
+                lazy: true
+                provider: users_in_memory
+
+                # activate different ways to authenticate
+                # https://symfony.com/doc/current/security.html#firewalls-authentication
+
+                # https://symfony.com/doc/current/security/impersonating_user.html
+                # switch_user: true
+
+        # Easy way to control access for large sections of your site
+        # Note: Only the *first* access control that matches will be used
+        access_control:
+            # - { path: ^/admin, roles: ROLE_ADMIN }
+            # - { path: ^/profile, roles: ROLE_USER }
+
+That's a mouthful of config! In the next three sections, the main 3
```

Comment:
`mouthful` sounds too informal to me. Will others agree?

Reply:
I guess Ryan and me often like adding some informal bits to the docs. I'm fine with removing it, although I do feel like the "Symfony brand" need a bit of informality every once in a while (e.g. the error ghost, the old SymfonyLive logo, the "Hey you!" badge, etc.).

(although, listing these things, it does seem like that's becoming a bit of an historic brand expression)